code/src/App.js

```diff
@@ -5,6 +5,7 @@ import { combineReducers, configureStore } from '@reduxjs/toolkit';
 import { Info } from './components/Info'
```

Comment:
I appreciate how clean your App.js is. If you wanted to make the newly entered tasks 'stick' on refresh, you could implement something called 'persisted state' in redux and expand on the reducer here. Here is an example:

`const reducer = combineReducers({
  tasks: tasks.reducer,
});

const persistedState = localStorage.getItem("reduxState")
  ? JSON.parse(localStorage.getItem("reduxState"))
  : {};

const store = configureStore({
  reducer,
  preloadedState: persistedState,
});

store.subscribe(() => {
  localStorage.setItem("reduxState", JSON.stringify(store.getState()));
}); `

The persisted state creates local storage for your store, so that every task entered in the 'store' aka the project, sticks to it and doesn't disappear on refresh.